index.rst

```diff
@@ -39,10 +39,12 @@ All aspects of the project that go through strict change control (e.g. mirror pr
 
 .. _Obs-Ops-Safety:
 
-Safety
-======
+Quick Safety Reference for Nighttime Personnel
```

Comment:
I think having "Safety" as the first word in the title would be better.. Maybe something like:


```
Safety: Quick Reference for Nighttime Personnel
```

or, preferably, in my opinion:

```
Safety: Quick Reference Guidelines
```